lib/compilers/WSL-CL.js

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2012-2017, Matt Godbolt
```

Comment:
You can probably this to your name if you're comfortable with it

Reply:
Yes, Matt told me this morning to do that but I missed it when merging in his changes. Thank you!